### ISSUE TITLE
twoway plot results in a gap between alignment lines and time series for small offset values

### ISSUE THREAD

**Issue by nicholaslivingstone** [DynamicTimeWarping/dtw-python#22]
* Python port of R's Comprehensive Dynamic Time Warp algorithm package version: 1.1.12
* Python version: 3.9.10
* Operating System: Windows 10

### Description

I'm working with timeseries data near zero. When trying to display a twoway plot of the alignment, I need to use small offset values to properly show alignments between the data. Small `offset` values seem to result in the alignment lines not fully touching the reference timeseries. 

### What I Did

A reproducible example which results in the following graph:  
```
x = np.linspace(0, 4 * np.pi)
y = np.sin(x) * 0.001
y2 = np.sin(x) * 2 * 0.001

alignment = dtw(y, y2, keep_internals=True)
alignment.plot(type='twoway', offset=-0.005)
```
![alignment offset](https://user-images.githubusercontent.com/35415469/180089478-c2a83b7b-a4a2-4604-948a-fab45dd5b50b.png)

As well as an example from the data I was working with which caused me to notice the issue. The alignment lines become especially skewed between the 600-800 index range. 

![alignment offset not reproducible](https://user-images.githubusercontent.com/35415469/180089730-aa46d2e4-327a-4664-8ebf-ebe5ca4cb9dd.png)

### Potential Fix
I think the issue might have something to do with some rounding or scaling in matplotlibs backend having to do with the twin axis. I've been able to more or less mitigate the issue by modifying the `dtwPlotTwoWay` function to plot both time series on the same axis instead of plotting on the twin axis.

```
def adjusted_dtwPlotTwoWay(...):
...

    ax.plot(times, yts - offset, **kwargs) # Plot on the same axis, adjust the time series by the offset

    if offset != 0:
        ql, qh = ax.get_ylim()
        ax2.set_ylim(ql + offset, qh + offset)

...
```
#### Results of the change
![alignment offset fixed](https://user-images.githubusercontent.com/35415469/180090905-bd392df9-7f38-45a1-ba6c-2ab5a3c223e7.png)

![alignment offset not reproducible fixed](https://user-images.githubusercontent.com/35415469/180090760-2fe846d1-22b4-4f52-bb28-5673a5b55e3a.png)

**Comment by tonigi** [comment]
Hi, thanks for the careful report and analysis. There might have a reason why i plotted on the secondary axis, but I forgot it. So I am inclined to merge your request. Could you please open a PR? Thanks

**Comment by tonigi** [comment]
Should be addressed by [e969187](https://github.com/DynamicTimeWarping/dtw-python/commit/e969187d8edbb2e9cc4e3e94422378d8507f0424)
